react/components/FilterSidebar.js

```diff
@@ -71,6 +73,14 @@ const FilterSidebar = ({
   // eslint-disable-next-line @typescript-eslint/no-use-before-define
   const currentTree = useCategoryTree(tree, categoryTreeOperations)
 
+  const selectedFacets = React.useMemo(() => { 
+    const filterKeys = filters.map(filter => filter.key);
+
+    return selectedFilters.filter((facet) => (
+      filterKeys.includes(facet.key) 
+      && facet.selected
+    )) }, [filters, selectedFilters]);
```

Comment:
Hi @igorbrasileiro , thanks for the reviews. 

Here I wanted to return an array of selected filters instead of just a number. I think it's more understandable this way, and allows further uses in the future if necessary.

Do you agree?

Reply:
Yeah, i agree, that the selectedFacets array value is good. So, wdyt about returning the `selectedFacets.length` inside the useMemo? By doing this you maintain with the selectedFacets, but only return the interested value, that's the length, if further changes need the selectedFacets the value is already calculated and only the return should change.

Reply:
```suggestion
    const filterKeys = filters.map(filter => filter.key);

    return selectedFilters.filter((facet) => (
      filterKeys.includes(facet.key) 
      && facet.selected
    )).length }, [filters, selectedFilters]);
```

Reply:
I'd still prefer to return the array, since it took me a while to understand what was going on with `selectedFilters`. An  explicit variable like this would have saved me some time.

But I decided to follow your suggestion, and added a comment on top of the `useMemo`.